services/web/client/source/class/qxapp/component/widget/NodePorts.js

```diff
@@ -37,19 +37,18 @@ qx.Class.define("qxapp.component.widget.NodePorts", {
    * @param isInputModel {Boolean} false for representing defaultInputs
    */
   construct: function(node, isInputModel = true) {
-    this._setLayout(new qx.ui.layout.Grow());
+    this.setIsInputModel(isInputModel);
+    this.setNode(node);
 
-    const label = new qx.ui.basic.Label();
-    node.bind("label", label, "value");
+    this._setLayout(new qx.ui.layout.Grow());
 
     const nodeUIPorts = this.__nodeUIPorts = new qx.ui.container.Composite(new qx.ui.layout.VBox()).set({
       appearance: "node-ports"
     });
 
-    this.setIsInputModel(isInputModel);
-    this.setNode(node);
+    this.base(arguments, node.getLabel(), nodeUIPorts);
```

Comment:
calling the super class 'in the middle' indicates quite a lot of knowledge about the way the superclass constructoir is implemented :) is this necessary?

Reply:
I wouldn't say it is necessary, I did it to test the reusability of the PanelView, and it work first go so we left it like that. The superclass constructor is our code, so we really know how its implemented. There is probably room for a small refactor here, but this is out of the scope of this PR, I think.